src/utils/runtime/src/logger.rs

```diff
@@ -388,7 +394,7 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
     // Tracing layer
     #[cfg(not(madsim))]
     if let Some(endpoint) = settings.tracing_endpoint {
-        println!("tracing enabled, exported to `{endpoint}`");
+        println!("opentelemetry tracing will be exported to `{endpoint}` if enabled");
```

Comment:
Why not `tracing::log`? I think `tracing::log` will work no matter `enable_tracing` is true or false

Reply:
`tracing-subscriber` is not inited until the very last line of this function. 🤣 No logs will be printed with `tracing::event` before that.